### PR TITLE
docs: add aks deployment examples in bot-sso-docker sample

### PR DESCRIPTION
**Pull request by hund030**

### CODE REVIEW

**Review by eriolchan — DISMISSED** [review]

File: bot-sso-docker/README.md [hunk 1]
```diff
@@ -89,6 +89,57 @@ This sample demonstrate how to containerize a Teams App and integrate the Docker
 - From TeamsFx CLI:
   1. Run command: `teamsfx preview --env dev`.
 
+## Deploying on Azure Kubernetes Service
+
+You can deploy a containerized Teams App on a Kubernetes cluster. This assumes that you have an existing Azure Kubernetes Service connected to your Azure Container Registry, which hosts your container images. If you do not have one, please refer to this tutorial to create one: [AKS Tutorials](https://learn.microsoft.com/azure/aks/tutorial-kubernetes-prepare-app).
```
Comment (Yimin-Jin) [inline bot-sso-docker/README.md:94]: If you just want users to create an Azure Kubernetes Service, wouldn't this link be better: https://learn.microsoft.com/zh-cn/azure/aks/learn/quick-kubernetes-deploy-cli

File: bot-sso-docker/README.md [hunk 2]
```diff
@@ -89,6 +89,57 @@ This sample demonstrate how to containerize a Teams App and integrate the Docker
...
+
+The following steps are provided as an example:
+
+1. Install dependency for setting up TLS. Refer to [Create an ingress controller](https://learn.microsoft.com/azure/aks/ingress-basic?tabs=azure-cli) and [Use TLS with Let's Encrypt certificates](https://learn.microsoft.com/azure/aks/ingress-tls?tabs=azure-cli#use-tls-with-lets-encrypt-certificates).
+    ```
+    helm install ingress-nginx ingress-nginx/ingress-nginx --create-namespace --namespace $NAMESPACE \
```
Comment (Yimin-Jin) [inline bot-sso-docker/README.md:100]: Executing this command directly will report an error, the steps should be:
1. `helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx`
2. `helm repo update`

Although there is a link to this step: [Create an ingress controller](https://learn.microsoft.com/azure/aks/ingress-basic?tabs=azure-cli), if you refer to that step exactly and then run `helm install` will report an error

File: bot-sso-docker/README.md [hunk 3]
```diff
@@ -89,6 +89,57 @@ This sample demonstrate how to containerize a Teams App and integrate the Docker
...
+        --set controller.nodeSelector."kubernetes\.io/os"=linux  \
+        --set defaultBackend.nodeSelector."kubernetes\.io/os"=linux  \
+        --set controller.healthStatus=true \
+        --set controller.service.externalTrafficPolicy=Local \
+        --set controller.service.annotations."service\.beta\.kubernetes\.io/azure-load-balancer-health-probe-request-path"=/healthz  \
+    helm install cert-manager jetstack/cert-manager --namespace $NAMESPACE --set installCRDs=true --set nodeSelector."kubernetes\.io/os"=linux
```
Comment (Yimin-Jin) [inline bot-sso-docker/README.md:106]: Same question here

File: bot-sso-docker/README.md [hunk 4]
```diff
@@ -89,6 +89,57 @@ This sample demonstrate how to containerize a Teams App and integrate the Docker
...
+The following steps are provided as an example:
+
+1. Install dependency for setting up TLS. Refer to [Create an ingress controller](https://learn.microsoft.com/azure/aks/ingress-basic?tabs=azure-cli) and [Use TLS with Let's Encrypt certificates](https://learn.microsoft.com/azure/aks/ingress-tls?tabs=azure-cli#use-tls-with-lets-encrypt-certificates).
+    ```
+    helm install ingress-nginx ingress-nginx/ingress-nginx --create-namespace --namespace $NAMESPACE \
+        --set controller.nodeSelector."kubernetes\.io/os"=linux  \
+        --set defaultBackend.nodeSelector."kubernetes\.io/os"=linux  \
+        --set controller.healthStatus=true \
+        --set controller.service.externalTrafficPolicy=Local \
+        --set controller.service.annotations."service\.beta\.kubernetes\.io/azure-load-balancer-health-probe-request-path"=/healthz  \
```
Comment (Yimin-Jin) [inline bot-sso-docker/README.md:105]: This code should be separated from the next line by deleting the "\" at the end.

**Pull request closed without merging**